UnityProject/Assets/ScriptableObjects/Traits/CommonTraitsSingleton.asset

```diff
@@ -42,6 +42,7 @@ MonoBehaviour:
   RollingPin: {fileID: 11400000, guid: 8e3c98655a1b54d4a9251d1b68a1ef1a, type: 2}
   LightReplacer: {fileID: 11400000, guid: 61d5dc37185ebac4ab0e92eb5ba2594f, type: 2}
   Pickaxe: {fileID: 11400000, guid: 1c4515f9903c29c4f8832bd2f32684ad, type: 2}
+  ScienceScan: {fileID: 0}
```

Comment:
Empty reference?

Reply:
You need to put the trait in the CommonTraits SO